.github/workflows/release.yml

```diff
@@ -70,5 +70,74 @@ jobs:
             wal-g-${{ matrix.db }}-${{ matrix.os }}-amd64.tar.gz.sha256
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-    env:
-      USE_LZO: 1
+
+  release-ubuntu-arm:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        db: [ pg, mysql, sqlserver, redis, mongo, fdb, gp ]
+        include:
+          - arch: aarch64
+            distro: ubuntu20.04
+      fail-fast: false
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Build on arch
+        uses: uraimo/run-on-arch-action@v2
+        with:
+          arch: ${{ matrix.arch }}
+          distro: ${{ matrix.distro }}
+          env: |
+            USE_LIBSODIUM: ${{ env.USE_LIBSODIUM }}
+            USE_LZO: ${{ env.USE_LZO }}
+          shell: /bin/bash
+          run: |
```

Comment:
Split this into multiple run commans with comments just like in the amd64 pipeline.